skia-bindings/build_support/skia_bindgen.rs

```diff
@@ -127,7 +131,7 @@ pub fn generate_bindings(build: &FinalBuildConfiguration, output_directory: &Pat
         .clang_args(&["-x", "c++"])
         .clang_arg("-v");
 
-    let target = cargo::target();
+    let target = target.unwrap_or_else(cargo::target);
```

Comment:
nit: I don't like this here, because it kind of defines a default "under the hood". It should be probably pulled up the call chain as far as possible so that `generate_bindings` only takes a `target: Target`, and not `target: Option<Target>`.

Reply:
Good point, that would be much cleaner. I'll change that.